src/server/utils/wallets/createSmartWallet.ts

```diff
@@ -0,0 +1,152 @@
+import { defineChain, type Address } from "thirdweb";
+import { smartWallet, type Account } from "thirdweb/wallets";
+import { createWalletDetails } from "../../../db/wallets/createWalletDetails";
+import { WalletType } from "../../../schema/wallet";
+import { thirdwebClient } from "../../../utils/sdk";
+import { splitAwsKmsArn } from "./awsKmsArn";
+import {
+  createAwsKmsKey,
+  type CreateAwsKmsWalletParams,
+} from "./createAwsKmsWallet";
+import {
+  createGcpKmsKey,
+  type CreateGcpKmsWalletParams,
+} from "./createGcpKmsWallet";
+import { createLocalWallet } from "./createLocalWallet";
+import { getAwsKmsAccount } from "./getAwsKmsAccount";
+import { getGcpKmsAccount } from "./getGcpKmsAccount";
+
+const getSmartWalletAddresses = async ({
```

Comment:
It seems unnecessary for this function to accept `accountFactoryAddress` only to have the exact same value in the response. Can we simplify to just return the smartAccountAddress?